client/src/app/core/repositories/base-repository.ts

```diff
@@ -403,6 +397,10 @@ export abstract class BaseRepository<V extends BaseViewModel, M extends BaseMode
         this.modifiedIdsSubject.next(modelIds);
     }
 
+    protected tapViewModels(viewModels: V[]): V[] {
```

Comment:
Must return void since it is only a tapping.

Reply:
Hmm... Then I will change the name. I want a function, that modifies the incoming value, because I use it in `AgendaItemRepository.service#200`. Maybe `pipe...`.